config/deploy.rb

```diff
@@ -43,7 +43,9 @@
 set :passenger_roles, :web
 set :sidekiq_systemd_role, :worker
 set :sidekiq_systemd_use_hooks, true
-set :bundler2_config_use_hook, true
+
+# bundler2 doesn't work on Ubuntu
+# set :bundler2_config_use_hook, true
```

Comment:
This is a showstopper for us. What are you seeing that isn't working?

Reply:
Basically, a bundle not found error:

```
00:07 bundler2:config
      01 bundle config --local deployment true
      01 bash: bundle: command not found
#<Thread:0x00007fa24829bca0@/Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/runners/parallel.rb:10 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
	9: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/runners/parallel.rb:12:in `block (2 levels) in execute'
	8: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:31:in `run'
	7: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:31:in `instance_exec'
	6: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/dlss-capistrano-3.11.1/lib/dlss/capistrano/tasks/bundle_config.rake:48:in `block (3 levels) in <top (required)>'
	5: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:80:in `execute'
	4: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:148:in `create_command_and_execute'
	3: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:148:in `tap'
	2: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/abstract.rb:148:in `block in create_command_and_execute'
	1: from /Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/backends/netssh.rb:170:in `execute_command'
/Users/azanella/.rvm/gems/ruby-2.6.3/gems/sshkit-1.21.2/lib/sshkit/command.rb:97:in `exit_status=': bundle config --local deployment true exit status: 127 (SSHKit::Command::Failed)
bundle config --local deployment true stdout: Nothing written
bundle config --local deployment true stderr: bash: bundle: command not found
```

Reply:
When you use RVM, did you run `gem install bundler` ?  I wonder if that `bundle config` command needs to be prefixed with the path to bundler in the RVM gemset.

Reply:
I tried manually installing bundler as the app user, and got the same result. I notice that the bundler2 step comes after bundler:config, which succeeds with the cap-rvm gem:

 ```
00:09 bundler:config
      01 /usr/local/rvm/bin/rvm default do bundle config --local deployment true
    ✔ 01 techmd@dor-techmd-worker-stage-b.stanford.edu 0.629s
00:10 bundler2:config
      01 bundle config --local deployment true
      01 bash: bundle: command not found
```

Without the cap-rvm gem, the bundler:config step fails first with:
```
00:08 bundler:config
      01 bundle config --local deployment true
      01 /usr/bin/env: ‘bundle’
      01 : No such file or directory
```

Reply:
Ah, this may not be important any longer since capistrano-bundler 2.0 supports bundler 2 and we don't need the custom hooks in dlss-capistrano any longer: https://github.com/sul-dlss/dlss-capistrano/pull/28